CHANGELOG.md

```diff
@@ -1,13 +1,24 @@
-vx.y.z - TBD
+v2.10.0 - 3 November 2021
 ------------
 
 Bug fixes:
 
-* [#247](https://github.com/godaddy/tartufo/issues/247) -- the `--branch` option
-  for `scan-remote-repo` has not worked since v2.0.2. Versions v2.2.0 through
-  v2.7.0 failed silently (not scanning the branch, and returning no error).
-  Versions v2.8.0 and later claimed the branch did not exist, even if it did.
-  This option now works correctly.
+* [#247](https://github.com/godaddy/tartufo/issues/247) All versions of tartufo from
+v2.2.0 through v2.9.0 inclusive mishandle `scan-remote-repo`. Only the repository's
+default branch was scanned, and secrets present in other branches would not be
+discovered. Additionally, the `--branch branch-name` option did not operate correctly
+for remote repositories. Some versions would scan nothing and report no errors, and
+other versions aborted immediately, claiming the branch did not exist (even if it did).
+v2.10.0 corrects these problems and may detect secrets that were not reported by previous versions.
```

Comment:
I'm not sure if it will have an impact on the Markdown formatting, but these lines should be indented by 2 spaces, to keep them aligned with their bullet point. At least, that is our convention. 😄